src/zenml/cli/stack_components.py

```diff
@@ -209,6 +209,21 @@ def register_stack_component_command(
             share: Share the stack with other users.
             args: Additional arguments to pass to the component.
         """
+        args = list(args)
```

Comment:
Can we get rid of this duplication here and in the update method by extracting this logic to a separate function?

Reply:
No good deed goes unpunished.. :) Yes, I moved everything inside the utils function (which required me to change some secrets-manager CLI code too).

Reply:
Haha 😄 thanks for updating it!